NuRadioReco/framework/event.py

```diff
@@ -562,14 +565,53 @@ def get_hybrid_information(self):
         """
         return self.__hybrid_information
 
+    def has_glitch(self):
+        """
+        Returns true if any channel in any station has a glitch
+        """
+        for station in self.get_stations():
+            for channel in station.iter_channels():
+                if channel.has_parameter(chp.glitch) and channel.get_parameter(chp.glitch):
+                    return True
+
+        return False
```

Comment:
I think we should implement this function also in utilities. Maybe in an RNO-G specific file, i.e., `rnog_station_utilities.py`?